src/runtime-rs/crates/resource/src/rootfs/block_rootfs.rs

```diff
@@ -46,18 +49,10 @@ impl BlockRootfs {
             ..Default::default()
         };
 
-        let device_id = d
-            .write()
-            .await
-            .new_device(&DeviceConfig::BlockCfg(block_device_config.clone()))
-            .await
-            .context("failed to create deviec")?;
-
-        d.write()
+        // create and insert block device into Kata VM
+        let device_info = do_handle_device(d, &DeviceConfig::BlockCfg(block_device_config.clone()))
```

Comment:
Is this part supposed to be in the `Device Manager refractor` commit?

Reply:
Yes, it will be the entry for handling device: new device, add device and get device info.